colearn_keras/keras_mnist.py

```diff
@@ -68,6 +70,37 @@ def prepare_data_loaders(location: str,
     return train_loader, vote_loader, test_loader
 
 
+# The dataloader needs to be registered before the models that reference it
+@FactoryRegistry.register_dataloader("KERAS_MNIST_WITH_DP")
```

Comment:
this is code duplication of @FactoryRegistry.register_dataloader("KERAS_MNIST") except for the `dp_enabled=True`. For simplicity, wouldn't it be better to re-use the `prepare_data_loaders` code?

Reply:
I need a separate loader so it's clear on UI side which I am selecting.

Reply:
done